frontend/server/workflow-helper.test.ts

```diff
@@ -83,7 +83,7 @@ describe('workflow-helper', () => {
       mockedGetPodLogs.mockResolvedValueOnce('pod logs');
 
       const stream = await getPodLogsStreamFromK8s('podName', 'namespace');
-      expect(mockedGetPodLogs).toBeCalledWith('podName', 'namespace');
+      expect(mockedGetPodLogs).toBeCalledWith('podName', 'namespace', undefined);
```

Comment:
Why is this mock called with ('podName', 'namespace', **undefined**) rather than ('podName', 'namespace', **'main'**)

Reply:
I guess default='main' is inside `getPodLogs`. when calling `getPodLogs` from `getPodLogsStreamFromK8s`, it doesn't specify the containerName. therefore, it's `undefined` here.

Reply:
To my understanding, the unspecified "optional" parameter can be `undefined`. It seems to me that if the `containerName` is not specified, the `getPodLogs` should be called with default value which is  **'main'** rather than `undefined`, because it is a **required** parameter. 

Would you mind changing the third parameter from `undefined` to **'main'** and see if the test still pass?

Reply:
let me having the default value for both `getPodLogsStreamFromK8s` and `getPodLogs` and update the test case to avoid the confusion. :-)